src/css/components/deployments.less

```diff
@@ -12,3 +12,10 @@
     width: 100%;
   }
 }
+
+.deployments {
+  td {
+    padding-bottom: @base-spacing-unit !important;
```

Comment:
Do we really want to use `!important` ? I does work but we could be more specific in the selector
```less
.deployments.table > tbody > tr {
  > td {
    padding-bottom: @base-spacing-unit;
    vertical-align: top;
  }
}
```